openmc/cell.py

```diff
@@ -197,7 +196,14 @@ def rotation(self):
 
     @property
     def rotation_matrix(self):
-        return self._rotation_matrix
+        if self.rotation is not None:
```

Comment:
You're right -- that's probably why I did it that way in the first place. I'll revert; thanks!